spec/requests/form_controller_spec.rb

```diff
@@ -141,6 +142,24 @@
           expect(response.body).to match("Review lettings log")
         end
       end
+
+      context "when viewing a user dependent page" do
+        context "when the dependency is met" do
+          let(:user) { FactoryBot.create(:user, :support) }
+
+          it "routes to the page" do
+            get "/logs/#{case_log.id}/organisation"
+            expect(response).to have_http_status(:ok)
+          end
+        end
+
+        context "when the dependency is not met" do
+          it "routes to the page" do
```

Comment:
I think this description is incorrect - they get redirected instead of routed to the page - at least based on the test